### PR TITLE
Update help for class option to match man pages

### LINKED ISSUE #3615: Document setting instance and base class in --help output
When opening Alacritty with `--class` option it changes the window instance instead the class.
For example, if I do `alacritty --class foo` and I check the window properties I got:
```
Instance: foo
Class: Alacritty
Title: Alacritty
```
`--title` option is working as expected, although.

### Infos

OS: Funtoo Linux
Window Manager: dwm
Alacritty: alacritty 0.5.0-dev (03bce99)

### PR DESCRIPTION
**Pull request by jeffwindsor**
Please make sure to document all user-facing changes in the `CHANGELOG.md` file.

Draft PRs are always welcome, though unless otherwise requested PRs will not be reviewed until all required and optional CI steps are successful and they have left the draft stage.

### CODE REVIEW

**Comment by jeffwindsor** [comment]
fixes #3615

File: alacritty/src/cli.rs [hunk 1]
```diff
@@ -137,7 +137,8 @@ impl Options {
                 Arg::with_name("class")
                     .long("class")
                     .takes_value(true)
-                    .help(&format!("Defines window class on Linux [default: {}]", DEFAULT_NAME)),
+                    .value_names(&["instance", "general"])
+                    .help(&format!("Defines window class hint on Linux [default: {},{}]", DEFAULT_NAME, DEFAULT_NAME)),
```
Comment (chrisduerr) [inline alacritty/src/cli.rs:141]: It looks like this only offers instance **and** general class as suggestions, however it should be clear that it is possible to specify just one of them.

The help output also instructs the user to use it as `--class instance general`, however it should be used as `--class instance,general`.

Comment (jeffwindsor) [reply alacritty/src/cli.rs:141]: Got it, let me look how to do that and update the PR.  BTW Glad to help

Comment (jeffwindsor) [reply alacritty/src/cli.rs]: @chrisduerr I was able to make the change to show
```
OPTIONS:
        --class <instance>,<general>               Defines window class hint on Linux [default: Alacritty,Alacritty]
```
is this correct or are we looking for below (matching man page)
```
OPTIONS:
        --class [ <instance> | <instance>,<general> ]               Defines window class hint on Linux [default: Alacritty,Alacritty]
```

Comment (chrisduerr) [reply alacritty/src/cli.rs]: You've fixed 50% of the existing problems. We want to match the manpage.

File: alacritty.yml [hunk 2]
```diff
@@ -83,8 +83,7 @@
   # Specifying '0' will disable scrolling.
   #history: 10000
 
-  # Number of lines the viewport will move for every line scrolled when
-  # scrollback is enabled (history > 0).
+  # Acceleration multiplier for mouse scrolling when scrollback is enabled (history > 0).
   #multiplier: 3
```
Comment (chrisduerr) [inline alacritty.yml:87]: This change seems unrelated?

Comment (jeffwindsor) [reply alacritty.yml:87]: man bad guys will pull that back out

File: alacritty/src/cli.rs (hunk 1 above)
Comment (jeffwindsor) [reply alacritty/src/cli.rs]: ok one min will try to get the other 50%

**Comment by jeffwindsor** [comment]
@chrisduerr, I am very new to rust so maybe I am not testing it in the same way.  I tested it manually by running `cargo run -- --help` on my branch and that produced
```
OPTIONS:
        --class <instance>|<instance>,<general>    Defines window class hint on Linux [default: Alacritty,Alacritty]
    -e, --command <command>...                     Command and args to execute (must be last argument)
```
And yes, I agree I am definitely bending the clap usage.  I was not able to find a way to use clap to create documentation for  instance OR instance, general like in the man page.

Any ideas would be appreciated.

**Comment by chrisduerr** [comment]
>  I tested it manually by running cargo run -- --help on my branch and that produced

You should test that it works too, not only that the help is correct.

File: CHANGELOG.md [hunk 3]
```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Regression in font rendering on macOS
 - Scroll down escape (`CSI Ps T`) incorrectly pulling lines from history
 - Dim escape (`CSI 2 m`) support for truecolor text
+- Docs for class in --help missing information on setting general class
```
Comment (kchibisov) [inline CHANGELOG.md:55]:
```suggestion
- Documentation for class in `--help` missing information on setting general class
```

File: extra/alacritty.man [hunk 4]
```diff
@@ -42,8 +42,8 @@ Increases the level of verbosity (the max level is \fB\-vvv\fR)
 Prints version information
 .SH "OPTIONS"
 .TP
-\fB\-\-class\fR [ <instance> | <instance>,<general> ]
-Defines the window class hint on Linux [default: Alacritty,Alacritty ]
+\fB\-\-class\fR <instance> | <instance>,<general>
+Defines the window class hint on Linux [default: Alacritty,Alacritty]
```
Comment (kchibisov) [inline extra/alacritty.man:46]: I feel like we should mention that on Wayland it sets `app_id` and in `--help` as well, since it's not something obvious and could make it clear for folks. Or even add `--app_id` cli argument(not sure that we really should though). What do you think?

File: alacritty/src/cli.rs [hunk 5]
```diff
@@ -122,7 +124,9 @@ impl Options {
             .arg(
                 Arg::with_name("class")
                     .long("class")
+                    .value_name("instance> | <instance>,<general")
```
Comment (kchibisov) [inline alacritty/src/cli.rs:127]:
```suggestion
                    .value_name("instance> | <instance>,<general>")
```

File: extra/alacritty.man (hunk 4 above)
Comment (chrisduerr) [reply extra/alacritty.man:46]: How does that work on Wayland? Only instance is supported and instance is app_id? General does nothing?

File: alacritty/src/cli.rs (hunk 5 above)
Comment (chrisduerr) [reply alacritty/src/cli.rs:127]: This is intentional, it's necessary since clap adds `<...>` around the value by itself with no way to turn that off as far as I'm aware. This is the best solution I could find (it's also why the `<` is missing at the start).

File: extra/alacritty.man (hunk 4 above)
Comment (kchibisov) [reply extra/alacritty.man:46]: I mean it's `app+id`, but yeah, only one thing exists, nothing more.

File: CHANGELOG.md [hunk 6]
```diff
@@ -53,6 +53,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Scroll down escape (`CSI Ps T`) incorrectly pulling lines from history
 - Dim escape (`CSI 2 m`) support for truecolor text
 - Incorrectly deleted lines when increasing width with a prompt wrapped using spaces
+- Docs for class in --help missing information on setting general class
+- Documentation for class in `--help` missing information on setting general class
```
Comment (kchibisov) [inline CHANGELOG.md:57]: borken merge/rebase,etc.

File: extra/alacritty.man [hunk 7]
```diff
@@ -42,8 +42,10 @@ Increases the level of verbosity (the max level is \fB\-vvv\fR)
...
+
+On Wayland the instance class sets the `app+id`, while the general class is ignored.
```
Comment (kchibisov) [inline extra/alacritty.man:48]: I'm sorry, I've pressed some magic button on my keyboard, it's `app_id`.

File: alacritty/src/cli.rs [hunk 8]
```diff
@@ -122,8 +124,10 @@ impl Options {
...
                     .takes_value(true)
-                    .help(&format!("Defines window class on Linux [default: {}]", DEFAULT_NAME)),
+                    .use_delimiter(true)
+                    .help(&format!("Defines window class or app+id on Linux [default: {}]", DEFAULT_NAME)),
```
Comment (kchibisov) [inline alacritty/src/cli.rs:130]: same here `app_id`, sorry.
